### PR TITLE
Explain sign-off requirements

### PR DESCRIPTION
**Pull request by yurishkuro**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -129,3 +130,65 @@ way to add such header is to run `make fmt`.
...
+    personal information I submit with it, including my sign-off) is
+    maintained indefinitely and may be redistributed consistent with
+    this project or the open source license(s) involved.
+```
+
+then you just add a line to every git commit message:
+
+    Signed-off-by: Joe Smith <joe@gmail.com>
+
+using your real name (sorry, no pseudonyms or anonymous contributions.)
```
Comment (yurishkuro) [inline CONTRIBUTING.md]: @caniszczyk I don't suppose the DCO-bot can enforce the "no pseudonyms" rule...

**Comment by pavolloffay** [comment]
@caniszczyk What is the policy for this? Can we merge PRs without `DCO`? Does it apply for all `jaegertracing` repositories?

File: CONTRIBUTING.md (hunk 1 above)
Comment (adriancole) [reply CONTRIBUTING.md]: fyi there are repos in opentracing that will need this, ex https://github.com/opentracing/opentracing-cpp there are a few who are using anonymous github handles intentionally or not

**Comment by yurishkuro** [comment]
@pavilloffay it will apply to all repos that we move to this org, but each of them will have this text.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -114,6 +114,7 @@ pull request is most likely to be accepted if it:
   review comments](https://github.com/golang/go/wiki/CodeReviewComments).
 * Has a [good commit
   message](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
+* Each commit must be signed by the author (see below).
```
Comment (vprithvi) [inline CONTRIBUTING.md]: Could you add a link to `see below`?

**Pull request merged**